src/index.js

```diff
@@ -0,0 +1,35 @@
+console.log("am a frog not a virus trust me ;)")
```

Comment:
`Console.log` kills the performance of an app please remove it.

Reply:
Do you mean this? i was kind sure was that something like this.
<img width="1280" alt="Screenshot 2019-09-20 at 16 59 40" src="https://user-images.githubusercontent.com/16047985/65358633-1728c200-dbc8-11e9-9297-5a5a8d976e61.png">